boot/freeldr/freeldr/lib/fs/iso.c

```diff
@@ -140,116 +171,151 @@ static ARC_STATUS IsoBufferDirectory(ULONG DeviceId, ULONG DirectoryStartSector,
     return ESUCCESS;
 }
 
-
 /*
  * IsoLookupFile()
  * This function searches the file system for the
  * specified filename and fills in an ISO_FILE_INFO structure
  * with info describing the file, etc. returns ARC error code
  */
-static ARC_STATUS IsoLookupFile(PCSTR FileName, ULONG DeviceId, PISO_FILE_INFO IsoFileInfoPointer)
+static ARC_STATUS IsoLookupFile(PCSTR FileName, ULONG DeviceId, PISO_FILE_INFO IsoFileInfo)
 {
-    UCHAR Buffer[SECTORSIZE];
-    PPVD Pvd = (PPVD)Buffer;
+    PCSTR FullPath = FileName;
+    PISO_VOLUME_INFO Volume;
+    PCSTR FileNameStr;
+    ULONG DirectoryPathLength;
     UINT32        i;
```

Comment:
@disean May I suggest you to remove the extra unneeded spaces between types and variables for the other ones in this function that you left untouched?

Reply:
Done.